app/apps/events/models.py

```diff
@@ -42,8 +42,17 @@ def event_values(self):
         Returns a dictionary with event values retrieved from Emitter object
         """
         event_values = self.emitter.__get_event_values__()
+        event_values.pop("variables", None)
         return event_values
 
+    @property
+    def event_variables(self):
+        """
+        Returns a dictionary with event values retrieved from Emitter object
```

Comment:
Think comment should be `Returns a dictionary with event variables retrieved from Emitter object`